samples/client/petstore/kotlin-allOff-discriminator/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.2.1-SNAPSHOT
\ No newline at end of file
+6.1.0-SNAPSHOT
```

Comment:
I've updated, but I have a problem with generating samples. 
It's failing on kotlin-default-values-*

Reply:
ok, I figured out what's the problem with the generator. I was running on Java 17 which failed to build the generator. 
Maybe it would be helpful to have it scripted or have this in a container with all JVM stuff preset.
Anyways. pushed the new samples with the right version :)